snuba/migrations/snuba_migrations/metrics/0001_metrics_buckets.py

```diff
@@ -1,65 +1,32 @@
 from typing import Sequence
 
-from snuba.clickhouse.columns import Array, Column, DateTime, Nested, String, UInt
```

Comment:
Was there a functional change to these existing migrations? Or were they just changed to use the templates?

Reply:
99% is using the template without functional changes.
Though I also swapped the order of two columns.